tldw_Server_API/app/api/v1/endpoints/writing_manuscripts.py

```
@@ -930,15 +866,1206 @@ async def delete_scene(
+
+@router.post(
+    "/scenes/{scene_id}/characters",
+    response_model=list[SceneCharacterLinkResponse],
+    status_code=status.HTTP_201_CREATED,
+    summary="Link a character to a scene",
+    tags=["manuscripts"],
+)
+async def link_scene_character(
+    scene_id: str,
+    payload: SceneCharacterLink,
+    db: CharactersRAGDB = Depends(get_chacha_db_for_user),
```

Ruff (0.15.9)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)